### ISSUE TITLE
[Github pages] Add organization home page

### ISSUE THREAD

**Issue by sebastian-nagel** [crawler-commons/crawler-commons#315]
We use the site https://crawler-commons.github.io/ to host API docs but there is no organization home page. Would be good have one, also if it's only to link or redirect (meta-refresh) to https://github.com/crawler-commons/.

**Comment by Chaiavi** [comment]
Good idea @sebastian-nagel, I saw it on many other projects but never delved into it.

I think we should have a formal page rather than a redirect
I will be happy to help with the page

**Comment by Chaiavi** [comment]
I didn't differentiate between the project and the organization pages, I am not sure what to put in the organization page

I would love to hear other's thoughts here
